src/components/AppSidebar/AppSidebar.vue

```diff
@@ -39,7 +39,7 @@
 				<!-- sidebar details -->
 				<div :class="{ 'app-sidebar-header__desc--with-star': canStar }" class="app-sidebar-header__desc">
 					<!-- favourite icon -->
-					<a v-if="canStar" :class="isStarred ? 'icon-starred' : 'icon-star'"
+					<a v-if="canStar" :class="[ isStarred ? 'icon-starred' : 'icon-star', { 'icon-loading-small': starLoading } ]"
```

Comment:
```suggestion
					<a v-if="canStar" :class="{ 'icon-starred': isStarred, 'icon-star': !isStarred, 'icon-loading-small': starLoading }"
```

is more readable IMO

Reply:
Hum, yes and no I'd say ^^
More readable for the array removed, but less readable because you use two times the same variable ^^

Reply:
yes, you see the variable twice because it is used to goggle two classes

Reply:
xD you don't say?
@korelstar Chose whatever you want :)

And let's merge this (also travis is fixed on master if you want to rebase :wink: )

Reply:
Okay, I chose the version from @ChristophWurst since I think it's more natural in vue.

Reply:
:rocket: :rocket: :rocket:

Reply:
😱😱😱😱

Reply:
I'm so sorry, @skjnldsv !

Reply:
:grin: